manifest.json

```diff
@@ -4,13 +4,13 @@
   "version": "0.1",
   "commands": {
     "listen-en": {
```

Comment:
Can you change this to `listen-en` and `listen-th` to `listen1` and `listen2` respectively?